pkgs/servers/teleport/generic.nix

```diff
@@ -154,7 +182,7 @@ buildGoModule rec {
   meta = with lib; {
     description = "Certificate authority and access plane for SSH, Kubernetes, web applications, and databases";
     homepage = "https://goteleport.com/";
-    license = licenses.asl20;
+    license = if lib.versionAtLeast version "15" then licenses.agpl3Plus else licenses.asl20;
```

Comment:
Teleport changed the license to AGPLv3 (or later) staring with version 15. https://goteleport.com/blog/teleport-oss-switches-to-agpl-v3/